microshift_release_notes/microshift-4-16-release-notes.adoc

```diff
@@ -0,0 +1,156 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="microshift-4-16-release-notes"]
+= {product-title} {product-version} release notes
+include::_attributes/attributes-microshift.adoc[]
+:context: release-notes
+
+toc::[]
+
+{product-title} provides developers and IT organizations with small-form-factor and edge computing delivered as an application that customers can deploy on top of their managed {op-system-base-full} devices at the edge. Built on {OCP} and Kubernetes, {microshift-short} provides an efficient way to operate single-node clusters in low-resource edge environments.
+
+{microshift-short} is designed to make control plane restarts economical and be lifecycle-managed as a single unit by the operating system. Updates, roll-backs, and configuration changes consist of simply staging another version in parallel and then - without relying on a network - flipping to and from that version and restarting.
+
+//TODO verify K8s version and another other relevant notes
+[id="microshift-4-16-about-this-release"]
+== About this release
+Version 4.16 of {product-title} includes new features and enhancements. {microshift-short} was introduced as Generally Available with {microshift-short} 4.14. Update to the latest version of {microshift-short} to receive all of the latest features, bug fixes, and security updates. This release uses link:https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.28.md[Kubernetes 1.28] with the CRI-O container runtime. New features, changes, and known issues that pertain to {microshift-short} are included in this topic.
+
+You can deploy {microshift-short} clusters to on-premise, cloud, disconnected, or offline environments.
+
+//TODO: Update OP system versions
+{microshift-short} {product-version} is supported on {op-system-ostree-first} and {op-system-base-full} 9.2 and 9.3.
+
+For lifecycle information, see the link:https://access.redhat.com/product-life-cycles?product=Red%20Hat%20build%20of%20Microshift,Red%20Hat%20Device%20Edge[{product-title} Life Cycle Policy].
+
+[id="microshift-4-16-new-features-and-enhancements"]
+== New features and enhancements
+
+This release adds improvements related to the following components and concepts.
```

Comment:
I know this is a standard RN text thing, but I wonder if shifting focus to the components and concepts helps this read better and creates cohesion with the heading. Maybe something like, `The following components and concepts are improved with this release.` I don't think there is anything wrong with the way it is now, to be clear. This is a style suggestion.

Reply:
It's a good point. We introduce passive voice reversing the structure, which is probably why the structure is the way it is now (guessing).

Reply:
Fair enough.